aws/middleware/middleware_test.go

```diff
@@ -187,3 +191,239 @@ func TestAttemptClockSkewHandler(t *testing.T) {
 		})
 	}
 }
+
+type HTTPClient interface {
```

Comment:
nit, idiomatically all of these test definitions are always unexported.

Reply:
changed